chsdi/models/vector/uvek.py

```diff
@@ -327,6 +327,46 @@ class Zaehstellenueber(Base, Vector):
 register('ch.astra.strassenverkehrszaehlung_messstellen-uebergeordnet', Zaehstellenueber)
 
 
+class Zaehlstellenuebergeordnet(Base, Vector):
+    __tablename__ = 'strassenverkehrszaehlung_uebergeordnet'
+    __table_args__ = ({'schema': 'astra', 'autoload': False})
+    __template__ = 'templates/htmlpopup/verkehrszaehlung_uebergeordnet.mako'
+    __bodId__ = 'ch.astra.strassenverkehrszaehlung-uebergeordnet'
+    __queryable_attributes__ = ['mlocname']
```

Comment:
according to mum this attribute is only supposed to be searchable in sphinx.
@AFoletti can you confirm that we dont neet object information / attribute search?

for the sphinx search you should set searchable=true in the bod